frontend/src/pages/Landing/Landing.tsx

```diff
@@ -14,7 +14,6 @@ export const Landing = ({ children }: { children: ReactNode }) => {
 
 	return (
 		<Box
-			backgroundColor="p12"
```

Comment:
this change broke the style of the login page. what pages is it necessary for?
![image](https://github.com/highlight/highlight/assets/1351531/c847fcf3-c9e5-42a0-aa4a-07757bf05663)